### PR TITLE
Manuscript and package response to reviews

### PR DESCRIPTION
**Pull request by lawinslow**
For your perusal @jread-usgs @aappling-usgs @sckott 
I will also email with details.

### CODE REVIEW

File: R/set_endpoint.R [hunk 1]
```diff
@@ -26,18 +26,15 @@
 set_endpoint = function(endpoint="production"){
 	
 	endpoint = tolower(endpoint)
+	match.arg(endpoint, c('production', 'prod', 'development', 'dev'))
```
Comment (aappling-usgs) [inline R/set_endpoint.R]: match.arg can do more than this code asks of it. i recommend this instead:
```r
set_endpoint = function(endpoint=c("production","development")){

  	endpoint = tolower(endpoint)
	endpoint <- match.arg(endpoint)

 	if(endpoint=="production"){
[etc.]
```

the above match.arg call should turn either 'prod' or 'production' (or 'pro', or 'p', or 'produc', etc.) into 'production'. same for development, all using partial matching. then the tests that follow only need to check for one value apiece. also, putting the options within the function declaration makes them part of the usage and, IMO, easier for users to find.

personally, i could do without line 28 altogether - i think it's reasonable to ask users to use lowercase based on the documented options available to them. but either way is OK with me.

**Comment by aappling-usgs** [comment]
besides that one comment above, this PR looks good to me :+1:

**Comment by lawinslow** [comment]
Thanks @aappling-usgs. Fixed.

**Comment by sckott** [comment]
went over changes, and reviewer comments - MS looks good to me

Were all reviewer suggestions about changes to the pkg code itself accounted for?  or still some to consider?

**Comment by lawinslow** [comment]
Thanks @sckott. All accounted for, other than some suggestions I'm pushing back on based on other co-author feedback.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Package: sbtools
 Title: USGS ScienceBase Tools
 Maintainer: Luke Winslow <lwinslow@usgs.gov>
-Version: 0.18.5
+Version: 0.19.2
```
Comment (jread-usgs) [inline DESCRIPTION]: You going to use 1.0.0 for CRAN (and manuscript) or keep < 1.0.0 for the initial? Either sounds fine to me.

File: R/item_get_wfs.R [hunk 3]
```diff
@@ -7,8 +7,10 @@
 #' @description 
 #' This function attempts to download the spatial layer data attached to the
 #' requested SB item. SB exposes discrete spatial objects (points, polygons) as 
-#' WFS web services. This requires the GDAL libraries \code{rgdal} and \code{gdalUtils} 
-#' to be installed, which may require the external GDAL libraries as well.
+#' web services based on the Open Geospatial Consortium, Web Feature Service (WFS)
```
Comment (jread-usgs) [inline R/item_get_wfs.R]: link out? http://www.opengeospatial.org/standards/wfs

**Comment by jread-usgs** [comment]
👍

File: DESCRIPTION (hunk 2 above)
Comment (lawinslow) [inline DESCRIPTION]: 1.0 for CRAN, once BAO ok's release.

File: R/item_get_wfs.R (hunk 3 above)
Comment (lawinslow) [inline R/item_get_wfs.R]: good idea, added.

**Pull request merged**